resource/repositories/operations.go

```diff
@@ -133,7 +133,7 @@ func GetResource(args GetResourceArgs) (resource.Resource, io.ReadCloser, error)
 		return resource.Resource{}, nil, errors.Trace(err)
 	}
 
-	res, reader, err = opRepo.set(data.Resource, data)
+	res, reader, err = opRepo.set(data.Resource, data, state.DoNotIncrementCharmModifiedVersion)
```

Comment:
Do we have a test that GetResource() does not update charm modified version?

Reply:
we don't have a test in this package at all hmm